### PR TITLE
Evaluation

### PR DESCRIPTION
**Pull request by n-young**
Most of the diff is moving files around; main changes are in `evaluation/generate_queries.py`. To use, first generate a workload using `load_small_data.sh` into a .txt file, then call the new script on it to generate queries for it. 100 queries are generated, labels are chosen uniformly at random, and metric values are chosen from a normal distribution around each metric's mean with variance mean/2. Metric ops are limited to the non-equality ones. See top lines for hyperparamaeters.

### CODE REVIEW

**Comment by n-young** [comment]
Addresses #21

**Review by desmondcheongzx — APPROVED** [review]
Looks good! In the future we'd have to think about how to generate more realistic queries (maybe possible to achieve with via the hyper-parameters you expose), but this will be very helpful while we try to get pass the goal line.
